### ISSUE TITLE
Documentation is hard to follow

### ISSUE THREAD

**Issue by mikku489** [glideapps/glide-data-grid#538]
Can you please share your source code for the excellent API story book? Just a snippet of the main part showing how you construct the grid, hook up callbacks, apply basic theming. There's a lot of great examples like "Add a column" with clean built-in support but I just can't seem to find your source code anywhere.

**Comment by sergeykimaia** [comment]
agreed, going through the storybook and seeing the examples, but not seeing how to achieve those results is annoying

**Comment by jassmith** [comment]
alright I will try to break this out. It was my desire because I mean... I write a lot of these, but yall convinced me.

**Comment by adamerose** [comment]
Being able to open each storybook example as a standalone CodeSandbox project would be great

**Comment by jassmith** [comment]
The problem with that is the data source. I have built a very fast "fake" data source that my mock data source doesn't serve as a factor in the performance evaluation. It's just not a good example of how a real data source should be built. This is useful for me for testing, less useful for docs...

**Comment by jassmith** [comment]
Okay the sources are now available. Not perfect but I consider this closed now. If you have other sugestions let me know in a new report.
